src/components/dataVisualizations/PieGraph.tsx

```diff
@@ -33,17 +27,16 @@ export const PieGraph = React.memo(function PieGraph({
                     label: dataLabel,
                     data: data,
                     fill: true,
-                    backgroundColor: backgroundColors
+                    backgroundColor: backgroundColors,
                 },
             ],
-        }
+        };
     }, [data, dataLabel]);
 
     return (
-        <>
+        <div style={{ flex: 1, display: "flex", width: "100%", height: "100%" }}>

```

Comment:
for this, do not update this wrapper. instead, update the wrapper of where you are using PieGraph.

The reason is that we don't want to force a user using pieGraph into these styles.

Reply:
My understanding is that this just makes piegraph scale with its box. When I did not have this wrapper and attempted to rescale piegraph, it did not cooperate

Reply:
if that's the case, we should
1. remove the empty <> </> tags 
2. put this style into the wrapper of where you are using pieGraph